examples/zone-printer/README.md

```diff
@@ -1,258 +1,252 @@
 
 ```shell
-for ctx in $(kubectl config get-contexts --kubeconfig=./zpkubeconfig -o name); do
+KUBECONFIG=cluster-1.yaml:cluster-2.yaml kubectl config view \
+  --flatten > clusters.yaml
+```
+
+## Step 2: Deploy the sample application
+
+Deploy the application along with its NodePort Service in each of the three
+clusters. You can get the cluster contexts from `kubectl config get-contexts`
+and iterate through all the clusters to deploy the application manifests. This
+could be accomplished by running the following loop:
+
+```shell
+for ctx in $(kubectl config get-contexts -o=name --kubeconfig clusters.yaml); do
   kubectl --context="${ctx}" create -f app/
 done
 ```
 
-## 2. Reserve a static IP on GCP.
+## Step 3: Reserve a static IP address
 
-We need to reserve a static IP on GCP for kubemci. Run the following
-command to reserve the IP.
+This command reserves a static IP address for the load balancer.
 
 ```shell
 ZP_KUBEMCI_IP="zp-kubemci-ip"
 gcloud compute addresses create --global "${ZP_KUBEMCI_IP}"
 ```
 
-## 3. Modify the Ingress manifest.
+## Step 4: Use the static IP on Ingress manifest
 
 Replace the value of `kubernetes.io/ingress.global-static-ip-name`
-annotation in `ingress/nginx.yaml` with the value of $ZP_KUBEMCI_IP.
+annotation in `ingress/nginx.yaml` with the value of `$ZP_KUBEMCI_IP`.
 
 ```shell
 sed -i -e "s/\$ZP_KUBEMCI_IP/${ZP_KUBEMCI_IP}/" ingress/nginx.yaml
 ```
 
-## 4. Deploy the Multi-Cluster Ingress (with kubemci)
+## Step 5: Deploy the multi-cluster Ingress with `kubemci`
 
-Run kubemci to create the multi-cluster ingress.
+Run `kubemci` to create the multi-cluster Ingress named `zone-printer`:
 
 ```shell
 kubemci create zone-printer \
     --ingress=ingress/nginx.yaml \
     --gcp-project=$PROJECT \
-    --kubeconfig=./zpkubeconfig
+    --kubeconfig=clusters.yaml
 ```
 
-Voila! You should have a multi-cluster ingress once the command
-successfully exits!
+This command creates the multi-cluster Ingress.
 
+## Step 6: View multi-cluster Ingress status
 
-## 5. Get the status of Multi-Cluster Ingress
-
-You can get the status of the multi-cluster ingress you just created
-by using `kubemci get-status` command.
+You can view the status of the multi-cluster Ingress you just created using the
+`kubemci get-status` command:
 
 ```shell
-kubemci get-status zone-printer --gcp-project=$PROJECT
+$ kubemci get-status zone-printer --gcp-project=$PROJECT
+
+Load balancer zone-printer has IPAddress 35.190.74.228 and
+is spread across 2 clusters (gke_yourproject_us-east4-a_cluster-1,
+gke_yourproject_europe-west1-c_cluster-2).
 ```
 
-## 6. Test the Multicluster Ingress
+## 6. Test the multi-cluster Ingress
 
-It may take several minutes for the ingress to propagate, but once it does, you
-can test it by curl'ing the IP from the get-status command. You should see it
-report a zone from the nearest region.
+It may take several minutes for the Cloud HTTP Load Balancers created through
```

Comment:
grammar seems off: maybe
It will take several minutes from when the Ingress and Cloud HTTP Load Balancer are created until they are ready.

Reply:
that makes sense. will fix.